torchvision/io/__init__.py

```diff
@@ -15,6 +15,8 @@
     write_video,
 )
 
+from .image import (read_png)
```

Comment:
Let's keep those private for now (i.e., not expose them in the `io` namespace), as I think we might want to expose instead a single high-level function for image decoding.